Docsp 16512 Conflict resolution edits (#1265)
mongodb/docs-realm: source/sync/conflict-resolution.txt

Conflicts arise when two or more users make changes to the same piece of
data independently. This can happen due to latency between device and
server or loss of connectivity. In this event, {+service-short+} automatically 
uses conflict resolution strategies to merge the changes. Specifically,
{+service-short+} handles conflict resolution using `operational transformation 
<https://docs.mongodb.com/realm/sync/protocol/#operational-transformation>`__,

> a function which provides specific rules that ensure all sides always converge 

Reviewer: ```suggestion
a set of rules that guarantee that all sides eventually converge
```

- OT does use functions but it's more of an implementation detail + may be confused with Realm functions, so let's leave that out
- This seems pretty similar to the next paragraph/sentence about eventual consistency - maybe we could combine them somehow?
